app/javascript/packs/userProfile.jsx

```diff
@@ -79,56 +134,176 @@ class UserProfile extends Component {
+                <button
+                  onClick={this.onClickEvent}
+                  className={
+                    this.state.showEvent
+                      ? "back-card-button button-color"
+                      : "btn-hover btn-color back-card-button"
+                  }
+                >
+                  <img className="button-logo-img" src="./profile/event.png" />
+                </button>
+
+                <button
+                  onClick={this.onClickAchi}
+                  className={
+                    this.state.showAchievement
+                      ? "back-card-button button-color"
+                      : "btn-hover btn-color back-card-button "
+                  }
+                >
+                  <img
+                    className="button-logo-img"
+                    src="./profile/achievements.png"
+                  />
+                </button>
+              </div>
+
               <button
                 className="card-button btn-hover btn-color"
-                onClick={this.handleClick}
+                onClick={this.handleClickFlip}
               >
                 Back
               </button>
             </div>
           </ReactCardFlip>
         </div>
+        {(this.state.showEvent || this.state.showAchievement) === false ? (
+          <div />
+        ) : (
+          this._renderSubComp()
+        )}
       </>
     );
   }
 }
 
+class Events extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      showOwnedEvents: false
+    };
+  }
+
+  changeList = () => {
+    this.setState({
+      showOwnedEvents: !this.state.showOwnedEvents
+    });
+  };
+
+  render() {
+    return (
+      <div className="toggle-list fade-in">
+        <h2 className="toggle-list-h2">Your Events</h2>
+        <button
+          className="card-button btn-hover btn-color card-button-event"
+          onClick={this.changeList}
+        >
+          {this.state.showOwnedEvents ? "created events" : "take a part"}
+        </button>
+        <div className="toggle-list-text">
+          <main className="st_viewport">
+            <div className="st_wrap_table" data-table_id="0">
+              <header className="st_table_header">
+                <div className="st_row">
+                  <div className="st_column _title">Title</div>
+                  <div className="st_column _event_time">Time</div>
+                  <div className="st_column _event_date">Date</div>
+                </div>
+              </header>
+
+              <div className="st_table">
+                {this.state.showOwnedEvents
+                  ? this.props.events.map(event => {
+                      return (
+                        <div className="st_row fade-in">
+                          <ReactTooltip />
+                          <div className="st_column _title">
+                            <a
+                              href={"http://localhost:3000/events/" + event.id}
```

Comment:
This is weird, how do you think this will behave on deploy? :thinking: Maybe do ``{`/events/${event.id}`}`` instead :question: